packages/app-js/src/types.ts

```diff
@@ -8,3 +8,17 @@ export type Log = {
   args: Array<any>,
   type: LogType
 };
+
+export type Snippet = {
+  text: string;
+  value: string;
+  code: string;
+  label?: object;
+};
+
+export interface CustomWindow extends Window {
+  api?: any;
```

Comment:
If we do this, no `any` (I've done some for Snipper where object is used in the PR into this)

Reply:
Removed from this branch in favour of implementing it in the api itself